package.json

```
@@ -1,9 +1,10 @@
 {
-  "name": "http-headers",
+  "name": "@readme/http-headers",
   "version": "1.0.0",
```

up to you here but if you wanted to do a pre-release version just to make sure it publishes properly:

```suggestion
  "version": "0.0.1",
```

---

Yeah, good call.